upgrade/satellite.py

```diff
@@ -21,7 +21,6 @@
 from upgrade.helpers.tasks import nonfm_upgrade
 from upgrade.helpers.tasks import post_migration_failure_fix
 from upgrade.helpers.tasks import pulp2_pulp3_migration
-from upgrade.helpers.tasks import pulp2_removal
```

Comment:
Could you please remove the `pulp2_removal` helper completely, since it won't be needed anymore?

Reply:
it can be used optionally 6.10 post upgrade, but I guess that is another branch `6.10.z` 
So we can remove from `master` completely 
Removing...